doc/manual/manual.tex

```diff
@@ -7860,12 +7856,18 @@ \subsubsection{Inner core convection}
   H,
 \end{align}
 
-where $Ra$ is the Rayleigh number. 
+where $Ra$ is the Rayleigh number and 6 is the 'source term', originated from the time-dependence of the reference temperature (secular cooling). In spherical geometry, $H=6$. 
```

Comment:
You probably mean: **H** is a source terms that originates...
Could you also add the fact that this is related to the nondimensionalization? Something like, 'Because we remove the adiabatic background temperature from the energy equation during non-dimensionalization, we have to add a source term that describes the decrease of the adiabatic temperature over time (secular cooling).